langchain4j-jlama/src/test/java/dev/langchain4j/model/jlama/JlamaLanguageModelIT.java

```diff
@@ -0,0 +1,48 @@
+package dev.langchain4j.model.jlama;
+
+import dev.langchain4j.model.language.LanguageModel;
+import dev.langchain4j.model.output.Response;
+import org.assertj.core.util.Files;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+import static dev.langchain4j.model.output.FinishReason.LENGTH;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JlamaLanguageModelIT {
+
+    static File tmpDir;
+
+    static LanguageModel model;
+
+
+    @BeforeAll
+    static void setup() {
+        tmpDir = Files.newTemporaryFolder();
+
+        model = JlamaLanguageModel.builder()
+                .modelName("openai-community/gpt2")
```

Comment:
minor: can we use the same model for chat and language models to avoid downloading extra models and make testsfaster?